comp/systray/systray/component.go

```diff
@@ -24,8 +19,3 @@ type Params struct {
 // Component for ddtray
 type Component interface {
 }
-
-// Module for ddtray
-var Module = fxutil.Component(
-	fx.Provide(newSystray),
-)
```

Comment:
The goal of this PR is to have a separate packages between Component and Module (See motivation from the PR description).

Reply:
Keeping the module here would be a good idea I think to simplify imports. But the `impl` should not be considered "private" as it can offers params, helpers, constructor for groups, ...